src/languages/en.ts

```diff
@@ -5287,14 +5285,16 @@ const translations = {
                     `changed the workspace to ${toPolicyName}${fromPolicyName ? ` (previously ${fromPolicyName})` : ''}`,
                 changeType: ({oldType, newType}: ChangeTypeParams) => `changed type from ${oldType} to ${newType}`,
                 delegateSubmit: ({delegateUser, originalManager}: DelegateSubmitParams) => `sent this report to ${delegateUser} since ${originalManager} is on vacation`,
-                exportedToCSV: `exported this report to CSV`,
+                exportedToCSV: `exported to CSV`,
                 exportedToIntegration: {
-                    automatic: ({label}: ExportedToIntegrationParams) => `exported this report to ${label}.`,
-                    automaticActionOne: ({label}: ExportedToIntegrationParams) => `automatically exported this report to ${label} via`,
-                    automaticActionTwo: 'accounting settings.',
+                    automatic: ({label}: ExportedToIntegrationParams) => `exported to ${label}`,
+                    automaticActionOne: ({label}: ExportedToIntegrationParams) => `exported to ${label} via`,
+                    automaticActionTwo: 'accounting settings',
                     manual: ({label}: ExportedToIntegrationParams) => `marked this report as manually exported to ${label}.`,
-                    reimburseableLink: 'View out-of-pocket expenses.',
-                    nonReimbursableLink: 'View company card expenses.',
+                    automaticActionThree: 'and successfully created a record for',
+                    and: 'and',
```

Comment:
Lets use 'and' from a different string. We can use or make a common string for this.